static_report/src/components/ComparisonReport/CRColumnDetail.tsx

```diff
@@ -95,10 +95,11 @@ export function CRColumnDetail({
         )}
       </GridItem>
 
+      {/* TODO: when columns detail page remove comment */}
       <GridItem>
-        <Flex alignItems="center">
```

Comment:
I've hidden the arrow icon because the detail page is still in progress. So this PR can merge without pending and users still can navigate to the table overview page.

<img width="884" alt="截圖 2022-08-25 08 46 15" src="https://user-images.githubusercontent.com/10325111/186548953-8717e73d-caba-4d44-b56c-a8293aa372c3.png">